pom.xml

```diff
@@ -1,311 +1,345 @@
-                    <artifactId>datanucleus-core</artifactId>
-                    <groupId>org.datanucleus</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>datanucleus-rdbms</artifactId>
-                    <groupId>org.datanucleus</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>bonecp</artifactId>
-                    <groupId>com.jolbox</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>calcite-core</artifactId>
-                    <groupId>org.apache.calcite</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>calcite-avatica</artifactId>
-                    <groupId>org.apache.calcite</groupId>
-                </exclusion>
-            </exclusions>
-        </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-service</artifactId>
+      <version>${hive.version}</version>
+      <exclusions>
+        <exclusion>
+          <artifactId>derby</artifactId>
+          <groupId>org.apache.derby</groupId>
+        </exclusion>
+        <exclusion>
+          <artifactId>datanucleus-api-jdo</artifactId>
+          <groupId>org.datanucleus</groupId>
+        </exclusion>
+        <exclusion>
+          <artifactId>datanucleus-core</artifactId>
+          <groupId>org.datanucleus</groupId>
+        </exclusion>
+        <exclusion>
+          <artifactId>datanucleus-rdbms</artifactId>
+          <groupId>org.datanucleus</groupId>
+        </exclusion>
+        <exclusion>
+          <artifactId>bonecp</artifactId>
+          <groupId>com.jolbox</groupId>
+        </exclusion>
+        <exclusion>
+          <artifactId>calcite-core</artifactId>
+          <groupId>org.apache.calcite</groupId>
+        </exclusion>
+        <exclusion>
+          <artifactId>calcite-avatica</artifactId>
+          <groupId>org.apache.calcite</groupId>
+        </exclusion>
+      </exclusions>
+    </dependency>
 
+    <!-- The latest versions of datanucleus seems to be faster than the one included in hive -->
+    <dependency>
+      <artifactId>datanucleus-api-jdo</artifactId>
+      <groupId>org.datanucleus</groupId>
+      <version>3.2.8</version>
+    </dependency>
 
-        <!-- The latest versions of datanucleus seems to be faster than the one included in hive -->
-        <dependency>
-            <artifactId>datanucleus-api-jdo</artifactId>
-            <groupId>org.datanucleus</groupId>
-            <version>3.2.8</version>
-        </dependency>
+    <dependency>
+      <artifactId>tez-dag</artifactId>
+      <groupId>org.apache.tez</groupId>
+      <version>${tez.version}</version>
+    </dependency>
 
+    <dependency>
+      <artifactId>tez-common</artifactId>
+      <groupId>org.apache.tez</groupId>
+      <version>${tez.version}</version>
+    </dependency>
 
-        <dependency>
```

Comment:
Nothing has changed for any of the non-build related dependencies, diff is just confused.